src/main/java/at/hannibal2/skyhanni/features/event/winter/ReindrakeWarpHelper.kt

```diff
@@ -0,0 +1,42 @@
+package at.hannibal2.skyhanni.features.event.winter
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object ReindrakeWarpHelper {
+
+    private val config get() = SkyHanniMod.feature.event.winter
+
+    private val patternGroup = RepoPattern.group("event.winter.reindrakewarphelper")
+
+    /**
+     * REGEX-TEST: §c§lWOAH! §cA §4Reindrake §cwas summoned from the depths!
+     */
+    private val spawnPattern by patternGroup.pattern(
+        "spawn.message",
+        "§c§lWOAH! §cA §4Reindrake §cwas summoned from the depths!",
+    )
+
+    @SubscribeEvent
+    fun onMessage(event: LorenzChatEvent) {
+        if (!isEnabled()) return
+        if (!spawnPattern.matches(event.message)) return
+        ChatUtils.clickToActionOrDisable(
+            "Click to warp to the Winter Island spawn!",
```

Comment:
the wording is duplicate for this function. i recomment something like "Reindrake warp helper detected a spawn!" or similar

Reply:
You still would want the string to say what you are clicking for right?
So `Someone fished up a Reindrake! Click to warp to the Winter Island spawn!`
or do you want it to only say "Reindrake warp helper detected a spawn"?

Reply:
oh yeah, should say that as well.

Reply:
you happy with the „someone fished up…“ suggestion?

Reply:
The thing is i have no idea if you get the announcement message if you fished it up yourself. If you fish it up yourself:
`A Reindrake forms from the depths.`

Reply:
Ok then the regex should also include that. If the message gets sent twice then, we can remove it again via repo pattern change

Reply:
so you want the message to trigger on both like this? 
`(WOAH…)|(A Reindrake…)`
and should the clickable message be: 
`A Reindrake was detected. Click to warp to the Winter Island spawn!`

Reply:
yes and yes

Reply:
you can just use the sea creature fished event for detecting when the player fishes their own reindrake

Reply:
as you can see in [this](https://www.youtube.com/watch?v=6QDkVlhYY40) video you still get the message when you fish it out yourself so we dont need to check if you fished it up yourself